projects/storefrontlib/src/cms-structure/guards/cms-page.guard.ts

```diff
@@ -23,7 +23,7 @@ export class CmsPageGuard implements CanActivate {
     protected cmsService: CmsService,
     protected protectedRoutesGuard: ProtectedRoutesGuard,
     protected service: CmsPageGuardService,
-    @Inject(Config) protected config: any
+    protected routingConfig: RoutingConfigService
```

Comment:
Migration for constructor?

Reply:
I don't think we need migration instructions for this class due to this reason. 
> No schematics are provided for the deprecated constructor of CmsPageGuard, because customers couldn't extend the constructor anyway (due to private dependencies before 2.0).
see https://github.com/SAP/spartacus/pull/6852#issue-385745472

Reply:
It was heavily refactored already for 2.0 and the docs cover it